### PR TITLE
update: eslint airbnb config established

### PR DESCRIPTION
**Pull request by rabbitwerks**
.editorconfig, .eslintrc added for eslint-airbnb and concise editor settings

### CODE REVIEW

**Comment by w3cj** [comment]
[Setup AirBnB Linting (with react plugin)](https://trello.com/c/crcsI0mL/34-setup-airbnb-linting-with-react-plugin)

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    'jest': true,
+  },
+  'rules': {
+    'no-use-before-define': 'off',
+    'react/jsx-filename-extension': 'off',
+    'react/prop-types': 'off',
+    'comma-dangle': 'off'
+  },
+  'globals': {
+    "fetch": false
```
Comment (w3cj) [inline .eslintrc.js]: React native has fetch. Lets change this to true.

File: .eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  'extends': 'airbnb',
```
Comment (w3cj) [inline .eslintrc.js]: Lets add the react recommended rules: https://github.com/yannickcr/eslint-plugin-react

File: .eslintrc.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  'parser': 'babel-eslint',
+  'env': {
+    'jest': true,
+  },
+  'rules': {
+    'no-use-before-define': 'off',
+    'react/jsx-filename-extension': 'off',
+    'react/prop-types': 'off',
```
Comment (w3cj) [inline .eslintrc.js]: Lets require prop-types. Makes it easier for others to know what props are required for a component.

File: .eslintrc.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  'extends': 'airbnb',
+  'parser': 'babel-eslint',
+  'env': {
+    'jest': true,
+  },
+  'rules': {
+    'no-use-before-define': 'off',
```
Comment (w3cj) [inline .eslintrc.js]: Why is this rule disabled? It usually is very handy for finding misspellings.

File: package.json [hunk 5]
```diff
@@ -4,7 +4,8 @@
     "start": "expo start",
     "android": "expo start --android",
     "ios": "expo start --ios",
-    "eject": "expo eject"
+    "lint": "eslint **/*.js",
+    "eject": "expo eject do not eject"
```
Comment (w3cj) [inline package.json]: We can just remove this to prevent people from ejecting.

**Review by w3cj — CHANGES_REQUESTED** [review]
See comments above.

File: .eslintrc.js (hunk 1 above)
Comment (rabbitwerks) [reply .eslintrc.js]: Okay, updating now.

File: package.json (hunk 5 above)
Comment (rabbitwerks) [reply package.json]: i will update the PR with the changes, thanks for the pointers CJ

File: .eslintrc.js (hunk 4 above)
Comment (rabbitwerks) [reply .eslintrc.js]: I was following an example of setting up eslint airbnb after initializing a project and they describe the settings as the bare minimum. will correct :)

**Review by w3cj — APPROVED** [review]

**Pull request merged**